### ISSUE TITLE
Does Open Shell cause a blank area on maximized windows on dual-monitor systems?

### ISSUE THREAD

**Issue by coffent** [Open-Shell/Open-Shell-Menu#654]
I have been plagued recently with a sporadic problem that has afflicted others with dual-monitor systems, where occasionally there is a blank area at the top of a maximized window that renders the window unusable.  See e.g. https://community.intel.com/t5/Graphics/Windows-10-multi-Monitor-maximized-window-blank-empty-space-on/m-p/623199#M73179.

I've been in contact with support people from Dell, the manufacturer of my computer, who say "We understand your concern and we are confident that this issue is caused by the Open shell application."

Have you found this problem?

Also, I don't know what version of Open Shell Menu I have.  When I click Help, I get the message "Can’t reach this page.  Make sure the web address //ieframe.dll/dnserrordiagoff.htm# is correct".  If I click "Check for Updates" I get the message "Failed to check for updates."

**Comment by bonzibudd** [comment]
I've never experienced this with dual monitors, but I suppose it's possible. Have you been able to reproduce it with Open-Shell uninstalled?

**Comment by coffent** [comment]
The problem is, I like and use Open Shell Menu, and I really don't want to uninstall it.  This problem occurs only infrequently (weeks) so I'd have to leave it uninstalled for up to several weeks to know for sure it was or wasn't the problem.  I am hoping someone might know for sure whether or not Open Shell Menu is the cause.

Also, if I do uninstall it (whatever version it is) and download & install the latest version, will I lose the customization I made with my current version?

**Comment by bonzibudd** [comment]
No, the customization should stay. But if you are reinstalling just to update to the latest, you can just run the installer without uninstalling the previous version.

**Comment by coffent** [comment]
Thanks - I've now downloaded/installed v. 4.4.160.  I'll see if that fixes the problem (if indeed it's caused by Open Shell).

BTW, Settings > Check for Updates now works, but I see that it's only for *language* updates, not program updates.  However clicking "Help" still gives the message "Can't reach this page:   //ieframe.dll/dnserrordiagoff.htm#" (Just FYI.)
